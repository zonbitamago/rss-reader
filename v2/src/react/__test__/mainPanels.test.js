'user strict';
jest.dontMock('../components/mainPanels.jsx');
import React from 'react';
import renderer from 'react-test-renderer';
import MainPanels from '../components/mainPanels.jsx';

test('samle', function() {
  expect(1).toBe(1);
});

test('snapshot', function() {
  const tree = renderer.create(
    <MainPanels/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
