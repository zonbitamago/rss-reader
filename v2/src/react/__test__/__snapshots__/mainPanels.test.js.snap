// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<section>
  mainPanels
</section>
`;
